epicure-frontend/src/pages/home/components/Footer/Footer.scss

```diff
@@ -1,16 +1,61 @@
 @import "../../../../data/types/typography.scss";
 @import "../../../../data/variables.scss";
+@import "../../../../data/UI/colors.scss";
 
-.homePage {
+
+footer {
     display: flex;
-    //flex-direction: column;
-    padding-left: 20px;
-    padding-right: 20px;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 100%;
+    height: 202px;
+    top: 3763px;
```

Comment:
why do u using a fix sizes?, it will effect your responsiveness

Reply:
done, used %